src/audio/src/src.c

```diff
@@ -27,6 +27,7 @@
 #include <ipc/control.h>
```

Comment:
Please add your name in the signed-off-name line like this:

``Signed-off-by: FirstName LastName <email>``